cpu/cc26x0/periph/rfc.c

```diff
@@ -101,16 +101,16 @@ uint16_t rfc_wait_cmd_done(void* ropCmd)
 
 void rfc_setup_ble(void)
 {
-    uint8_t buf[sizeof(radio_setup_t) + 3];
+    /*uint8_t buf[sizeof(radio_setup_t) + 3];
     radio_setup_t *rs = (radio_setup_t *)((uintptr_t)(buf + 3) & (0xFFFFFFFC));
     memset(rs, 0, sizeof(rs));
 
-    //run_command_ptr(&rs->op);
+    run_command_ptr(&rs->op);*/
```

Comment:
I've been trying to get the setup command to work for BLE but I was unsuccessful up until now. I'll try again this afternoon.